sessions/test_cookies.py

```diff
@@ -580,7 +579,7 @@ class StickyCookieConfig(tester.TempestaTest):
     @dmesg.unlimited_rate_on_tempesta_node
     def check_cannot_start_impl(self, msg):
         self.oops_ignore = ["WARNING", "ERROR"]
-        with self.assertRaises(CmdError, msg=""):
+        with self.assertRaises(error.BaseCmdException, msg=""):
```

Comment:
We should expect only `ProcessBadExitStatusException` here. Other exceptions are an unexpected result.